apps/communications/contacts/test/marionette/list_test.js

```diff
@@ -51,6 +51,68 @@ marionette('Contacts > List', function() {
     });
   });
 
+  suite('List Order > ', function() {
+    var data = [['GG', 'E'], ['AA', 'Z'],
+      ['XX', 'C'], ['CC', 'X'], ['EE', 'G'], ['FF', 'F'], ['HH', 'D'],
+      ['BB', 'Y'], ['YY', 'B'], ['ZZ', 'A'], ['DD', 'H']];
+    setup(function() {
+      data.forEach(contactData => {
+        subject.addContact({
+          givenName: contactData[0],
+          familyName: contactData[1]
+        });
+      });
+    });
+
+    function getNamesInList() {
```

Comment:
I realised thatwe need more than the marionette api, since we don't need the elements but it contents. So will move the method to `contacts`